pgxn/neon/pagestore_smgr.c

```diff
@@ -293,7 +293,6 @@ neon_wallog_pagev(SMgrRelation reln, ForkNumber forknum, BlockNumber blocknum,
 								blkno,
 								RelFileInfoFmt(InfoFromSMgrRel(reln)),
 								forknum)));
-				Assert(false);
```

Comment:
The comment above says "So in a standby, make this an assertion", so maybe this Assert should stay in place?

Reply:
Restored